mne/preprocessing/_optical_density.py

```diff
@@ -0,0 +1,47 @@
+# Authors: Robert Luke <mail@robertluke.net>
+#          Eric Larson <larson.eric.d@gmail.com>
+#
+# License: BSD (3-clause)
+
+from ..io import BaseRaw
+from ..io.constants import FIFF
+from ..utils import _validate_type, warn
+from ..io.pick import _picks_to_idx
+
+import numpy as np
+
+
+def optical_density(raw):
+    r"""Convert NIRS raw data to optical density.
+
+    Parameters
+    ----------
+    raw : instance of Raw
+        The raw data.
+
+    Returns
+    -------
+    raw : instance of Raw
+        The modified raw instance.
+
+    """
+    raw = raw.copy()
```

Comment:
Yes you can do .load_dara() at the end of the copy then you know raw._data is the

Reply:
Thanks